pages/vi/vi-github-and-repositories.md

```diff
@@ -164,14 +164,14 @@ The **upstream repository** is the one we are contributing to.
 #### Configure the upstream repository to your fork
 
 1. `cd <YourUserName>.github.io.`
-2. `git remote -v` see above to make sure you are pushing and fetching to your own repository on GitHub as the origin
+2. `git remote -v` see above to make sure you are pushing and fetching to your own repository on GitHub as the origin([more info](https://git-scm.com/docs/git-checkout))
```

Comment:
looks like you've inversed the links.

This should be the link for remote instead